node/router/src/lib.rs

```diff
@@ -81,7 +81,7 @@ pub struct InnerRouter<N: Network> {
     /// The cache.
     cache: Cache<N>,
     /// The resolver.
-    resolver: Resolver,
```

Comment:
Is the resolver only used in the `router` crate? We currently make it public (see the `pub use helpers::*;` above).
If it is only used, and intended to be used, here, we should reduce its visibility.